index.js

```diff
@@ -250,12 +272,16 @@ DB.prototype.get = function (key, opts, cb) {
   }
   this.kv.get(key, opts, function (err, docs) {
     if (err) return cb(err)
-    // Filter deletions & map to old expected format.
-    docs = filterObj(docs, function (key, value) {
-      return !value.deleted
-    })
-    docs = mapObj(docs, function (key, value) {
-      return value.value
+    docs = mapObj(docs, function (version, value) {
```

Comment:
Let's pass `opts = {fields: true}` to hyperkv and merge `v` into the deleted doc. Also, let's call it `doc` in the map function not `value` - we have enough things called `value` being passed around!